2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,45 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
+
+    let liElement = document.createElement("li");
+    liElement.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+    list.appendChild(liElement);
+    liElement.innerText = todo.task;
+
+
+
+    let spanElement = document.createElement("span");
+    spanElement.className = "badge bg-primary rounded-pill";
+    liElement.appendChild(spanElement);
+
+
+    let includeIcon = document.createElement("i");
+    includeIcon.className = "fa fa-check";
+    includeIcon.setAttribute("aria-hidden", true);
+    spanElement.appendChild(includeIcon);
+
+    let removeIcon = document.createElement("i");
+    removeIcon.className = "fa fa-trash";
+    removeIcon.setAttribute("aria-hidden", true);
+    spanElement.appendChild(removeIcon);
+
+
+    includeIcon.addEventListener("click", () => {
+      if (liElement.style.textDecoration === "line-through") {
```

Comment:
Good. This updates the style of the selected ToDo item. Do you think we should also update the actual list/array?